google/cloud/pubsub/internal/defaults.cc

```diff
@@ -25,16 +32,59 @@ namespace pubsub_internal {
 inline namespace GOOGLE_CLOUD_CPP_PUBSUB_NS {
 
 using std::chrono::seconds;
+using ms = std::chrono::milliseconds;
 
-std::size_t DefaultMaxConcurrency() {
-  auto constexpr kDefaultMaxConcurrency = 4;
+std::size_t DefaultThreadCount() {
+  auto constexpr kDefaultThreadCount = 4;
   auto const n = std::thread::hardware_concurrency();
-  return n == 0 ? kDefaultMaxConcurrency : n;
+  return n == 0 ? kDefaultThreadCount : n;
+}
+
+Options DefaultCommonOptions(Options opts) {
+  auto emulator = internal::GetEnv("PUBSUB_EMULATOR_HOST");
+  if (emulator.has_value()) {
+    opts.set<EndpointOption>(*emulator).set<GrpcCredentialOption>(
+        grpc::InsecureChannelCredentials());
+  }
```

Comment:
Perhaps move this next to the other setting for `GrpcCredentialOption`?

Reply:
the `EndpointOption`s are colocated 🤷‍♂️